hugo/layouts/partials/how-works.html

```diff
@@ -0,0 +1,33 @@
+<section id="how-works" class="how-for-you pb-5">
+    <div class="container text-dark pt-5">
+        <header class="text-center mb-5">
+            <h2>How source{d} works for you</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-4 mb-5">
+                <div class="text-center mb-4">
+                    <img src="img/icons/engineers-and-researchers.svg" alt="Engineers & Researchers" height="150" width="150">
+                </div>
+                <h3 class="title-line line-yellow">Engineers & Researchers</h3>
+                <p class="mb-4">From data retrieval, language analysis and machine learning on code tools, datasets and models, the source{d} stack can be helpful for many software engineering use cases.</p>
+                <a href="get-started" class="btn btn-highlight w-100">Learn More</a>
```

Comment:
same as in https://github.com/src-d/landing/pull/399#discussion_r287326996

The three links point to the very same page and section;
I wonder if it should open its dedicated section depending on your interests: research, management or business.

Reply:
The three buttons are still pointing to the very same point: `get-started`
Does it make sense @vcoisne ?
![image](https://user-images.githubusercontent.com/2437584/59177543-b7e4fb80-8b5c-11e9-807b-923f9c7d8755.png)

Reply:
imo, these three links should be as done in https://github.com/src-d/landing/pull/405/commits/9f5392c1000fca0a1acdaf3cdbab83d0dd5b952a
- `get-started#engineers-tab`
- `get-started#managers-tab`
- `get-started#executives-tab`